config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml

```diff
@@ -9,3 +9,20 @@ presubmits:
         command:
         - make
         - test
+  - name: pull-kube-storage-version-migrator-e2e
+    decorate: true
+    always_run: true
+    # TODO: remove this when the test is stable
+    optional: true
+    labels:
+      # are these necessary? what are they for?
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181019-3f9819d46-master
+        args:
+        - "--upload=gs://kubernetes-jenkins/logs"
+        - --
+        - --extract=ci/latest
+        - --test-cmd=../test/e2e/test-cmd.sh
```

Comment:
(you will still need other flags to bring up a cluster, just copy from the [gce job](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml#L125-L148)?)

Reply:
Done.

Reply:
add a timeout?